slack_sdk/socket_mode/builtin/client.py

```diff
@@ -67,7 +67,7 @@ def __init__(
         trace_enabled: bool = False,
         all_message_trace_enabled: bool = False,
         ping_pong_trace_enabled: bool = False,
-        ping_interval: float = 10,
+        ping_interval: float = 5,
```

Comment:
It looks like `@slack/socket-mode` uses a 30s timeout. Is it worthwhile for us to decrease it's timeout as well?
https://github.com/slackapi/node-slack-sdk/blob/main/packages/socket-mode/src/SocketModeClient.ts#L328

Reply:
@mwbrooks Good point. This Python client uses 20 seconds (it uses 5 sec interval * 4 in the check_state method). I think there is no single right answer for this but 15 or 20 seconds may be better to reduce idle time a bit. Before making the change, we can see how it affects (the behavior could depend on the underlying WS client implementation).